src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/model/translator.kt

```diff
@@ -8,7 +8,7 @@ import uk.gov.justice.digital.hmpps.prisonersearchindexer.services.canonicalPNCN
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.services.canonicalPNCNumberShort
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.services.dto.nomis.OffenderBooking
 
-fun Prisoner.translate(existingPrisoner: Prisoner? = null, ob: OffenderBooking, incentiveLevel: Result<IncentiveLevel?>, restrictedPatientData: RestrictedPatient?): Prisoner {
+fun Prisoner.translate(existingPrisoner: Prisoner? = null, ob: OffenderBooking, incentiveLevel: Result<IncentiveLevel?>, restrictedPatientData: Result<RestrictedPatient?>): Prisoner {
```

Comment:
this was odd previously - we coped with incentives failing, but not restricted patients